definitions/npm/@testing-library/react_v9.x.x/flow_v0.104.x-/react_v9.x.x.js

```diff
@@ -115,35 +115,28 @@ declare module '@testing-library/react' {
     eventProperties?: TInit
   ) => boolean;
 
-  declare type Queries = { ... };
-
-  declare type RenderResult<Q: Queries = GetsAndQueries> = {|
+  declare type RenderResult = {|
     container: HTMLDivElement,
     unmount: () => void,
     baseElement: HTMLElement,
     asFragment: () => DocumentFragment,
     debug: (baseElement?: HTMLElement) => void,
     rerender: (ui: React$Element<*>) => void,
-  |} & Q;
+  |} & GetsAndQueries;
 
-  declare export type RenderOptions<Q: Queries = { ... }> = {|
+  declare export type RenderOptions = {|
     container?: HTMLElement,
     baseElement?: HTMLElement,
     hydrate?: boolean,
-    queries?: Q,
+    queries?: any,
```

Comment:
If we want to get rid of this `any` here, help is appreciated over https://github.com/flow-typed/flow-typed/pull/3705